### PR TITLE
Version 3.2.4

### LINKED ISSUE #2142: file sharing error
Nicotine+ Version: 3.2.3
GTK Version: 3.24.34
Python Version: 3.10.5 (main, Aug  5 2022, 05:55:00)  [GCC 12.1.0 64 bit (AMD64)]

Type: <class 'ValueError'>
Value: list.remove(x): x not in list
Traceback:   File "D:/a/nicotine-plus/nicotine-plus/pynicotine/gtkgui/dialogs/preferences.py", line 629, in cell_toggle_callback
  File "D:/a/nicotine-plus/nicotine-plus/pynicotine/gtkgui/dialogs/preferences.py", line 620, in set_shared_dir_buddy_only

### PR DESCRIPTION
**Pull request by mathiascode**
Closes #2142

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 # Release Notes
 
-## Version 3.2.4 (Release Candidate 1)
+## Version 3.2.4 (August 6, 2022)
 
 ### Corrections
 
  * Fixed a regression where shared folders could not be added using the Preferences dialog
  * Fixed a performance regression when updating file transfer lists
+ * Fixed a potential crash when downloading many large files at once
```
Comment (slook) [inline NEWS.md:9]: ` * Fixed a potential buffer overflow crash when displaying the estimated time of a very large download`

File: NEWS.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
 # Release Notes
 
-## Version 3.2.4 (Release Candidate 1)
+## Version 3.2.4 (August 6, 2022)
 
 ### Corrections
 
  * Fixed a regression where shared folders could not be added using the Preferences dialog
  * Fixed a performance regression when updating file transfer lists
```
Comment (slook) [inline NEWS.md:8]: ... `on some systems`

Comment (mathiascode) [reply NEWS.md:8]: As far as I can tell, it affects all systems, including mine, when multiple transfers are active.

File: NEWS.md (hunk 1 above)
Comment (mathiascode) [reply NEWS.md:9]: buffer overflow is not relevant information for an average user

**Pull request merged**